src/luxor_overrides.jl

```diff
@@ -273,3 +274,253 @@ function background(background_color)
     end
     Luxor.background(background_color)
 end
+
+"""
+    apply_transform(transform::Vector{Float64} , poly::Vector{Point})
+
+applies the transform , got by getmatrix() on every point in the poly
+and returns a new poly.
+
+move this to luxor_overrides_util.jl later.
+"""
+function apply_transform(transform::Vector{Float64}, poly::Vector{Point})
+    retpoly = Point[]
+    for pt in poly
+        res = cairotojuliamatrix(transform) * [pt[1], pt[2], 1]
+        push!(retpoly, Point(res[1], res[2]))
+    end
+    retpoly
+end
+
+"""
+updates CURRENT_JPATHS
+if used inside strokepath() pass action=:stroke
+and inside fillpath pass :fill.
+move this to luxor_overrides_util.jl later.
+"""
+function update_currentjpath(action::Symbol)
+    if CURRENT_FETCHPATH_STATE == true
```

Comment:
Hi @ArbitRandomUser I think you wrote smth like this a couple of time shouldn't in general
`if a == true` be the same as `if a` and `if a == false` the same as `if !a`